testpmd/hooks/templates/sriov-policy.yml.j2

```
@@ -0,0 +1,35 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: {{ sriov['policy'] }}
+  namespace: openshift-sriov-network-operator
+spec:
+  nodeSelector:
+    {{ sriov['nodeselector'] | default('"node-role.kubernetes.io/worker": ""') }}
+  resourceName: {{ sriov['resource'] }}
+  priority: {{ sriov['priority'] | default(99) }}
+{% if sriov['mtu'] is defined %}
+  mtu: {{ sriov['mtu'] }}
+{% endif %}
+  numVfs: {{ sriov['numvfs'] }}
+  nicSelector:
+{% if sriov['vendor'] is defined %}
+    vendor: {{ sriov['vendor'] }}
+{% endif %}
+{% if sriov['deviceid'] is defined %}
+    deviceID: {{ sriov['deviceid'] }}
+{% endif %}
+{% if sriov['rootdevices'] is defined %}
+    rootDevices:
+{% for rootdevice in sriov['rootdevices'] %}
+      - {{ rootdevice }}
+{% endfor %}
+{% endif %}
+    pfNames:
+{% for device in sriov['devices'] %}
+      - {{ device }}
+{% endfor %}
+  deviceType: {{ sriov['device_type'] | default('netdevice') }}
+{% if sriov['isrdma'] is defined %}
+    isRdma: {{ sriov['isrdma'] }}
+{% endif %}
```

This was not before, but I think the indentation is not as expected per the description of the SRIOV Net Node policy, also I see this is only true or false, so it could be set as this:
```suggestion
  isRdma: {{ sriov['isrdma']|default(false)|bool }}
```

---

Correct, this was a last minute change, in case we want to use Mellanox cards (Casa might do this), I'll include your suggestion.